From 5cb72982b130f2d1d598ed4669c3ea9cbca37bb7 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Tue, 16 Oct 2018 14:24:47 +0300
Subject: [PATCH 329/706] enetc_tsn: Replace obsolete xmit_cbdr return values

Fixes: ef58169b54e3f651f19252e053338f511aae9f9f ("enetc: Cleanup CBDR
code, drop unused async support and fix coverity issue")

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 2e26c7e36c80affbab3c00cb1ade097c1db8c15e)
(cherry picked from commit 9b167c5d83e29b44b61fe33d88257497580f4889)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc_tsn.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc_tsn.c b/drivers/net/ethernet/freescale/enetc/enetc_tsn.c
index 47457e02fc36..f8308b4fcbdf 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_tsn.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc_tsn.c
@@ -78,7 +78,7 @@ static int xmit_cbdr(struct enetc_si *si, int i)
 	} while (timeout);
 
 	if (!timeout)
-		return ENETC_CMD_TIMEOUT;
+		return -EBUSY;
 #if 0
 	enetc_clean_cbdr(si);
 #endif
@@ -96,7 +96,7 @@ static int xmit_cbdr(struct enetc_si *si, int i)
 
 	ring->next_to_clean = nc;
 
-	return ENETC_CMD_OK;
+	return 0;
 }
 
 /* Class 10: Flow Meter Instance Statistics Query Descriptor - Long Format */
-- 
2.17.1

